app/database/DBConnection.java

```diff
@@ -0,0 +1,89 @@
+package database;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+
+/**
+ * Created by HuyNguyen on 4/3/17.
+ */
+public class DBConnection {
+  Connection con;
+
+  /**
+   * Forms connection to a database given a username and password. The url will be hardcoded to the database and currently only acts on localhost:3306.
+   * Auto commit is also turned off in this method.
+   *
+   * @param username String username
+   * @param password String password
+   * @param database String database to connect to (for this project, it will either be rbttutor or rbttutorunittesting
+   * @return         Connection object of the database connection
+   */
+  public Connection formConnection(String username, String password, String database) {
+    try {
+      String url = "jdbc:sqlite:./chatapp.db";
```

Comment:
There's already a database used in the chat server -- is there a way to make this more stateless?